### DISCUSSION TITLE
Browser and Desktop Versions not Syncing

### DISCUSSION THREAD

**Discussion by ghost** [community/community#74981] [New to GitHub]
Hello,

I am very new to GitHub, and I am trying to make a game using Unreal Engine 5 with him. We are using GitHub to collaborate. We are running into an issue when I commit, he doesn't see the commit in his desktop app. However, he does see the commit in the browser version.

Is there any way to get the two version of GitHub to sync with one another so he doesn't have to re-download the code from the browser version every single time?

Thanks!

**Comment by MegatIrfan** [comment]
Hi  :) 

Here are some steps you and your collaborator can take to troubleshoot and potentially resolve the issue:

**1. Check Branches:**
Ensure that both of you are working on the same branch. GitHub Desktop might not automatically switch branches, so it's crucial to confirm that you are both committing to and pulling from the same branch.

**2. Sync/Pull in GitHub Desktop:**
After you commit changes using GitHub Desktop, try clicking on the "Sync" button in the top-right corner. This will pull changes from the remote repository to your local repository. Your collaborator should also do this to ensure their local repository is up to date.

**3. Fetch Origin in GitHub Desktop:**
In GitHub Desktop, there should be an option to fetch from or push to the origin. Ensure both of you are fetching changes from the origin to update your local branches with changes made by your collaborator.

**4. Check Remote URL:**
Confirm that the remote URL in your local repository settings matches the correct GitHub repository. This can be found in GitHub Desktop by going to Repository > Repository Settings > Remote.

**5. Use Command Line:**

If the GitHub Desktop app is not behaving as expected, try using the Git command line. Open a terminal or command prompt and navigate to your local repository. Use commands like git pull origin <branch> to pull changes.

**6. Reinstall GitHub Desktop:**
If the issue persists, consider reinstalling GitHub Desktop to ensure you have the latest version, and there are no corrupted files causing the problem.

**7. Update Unreal Engine .gitignore:**

Ensure that the .gitignore file for your Unreal Engine project is correctly set up to ignore unnecessary files. This can prevent unexpected issues with version control.

**8. Check Permissions:**
Make sure both collaborators have the necessary permissions to push and pull changes from the repository.

**9. Verify Commit Author Email:**
Verify that the email address associated with your commits matches the email associated with your GitHub account. Mismatched email addresses can cause issues with commit visibility.

**10. Review GitHub Desktop Documentation:**
Check the GitHub Desktop documentation for any specific troubleshooting steps related to your issue.
Additional Tips:
Regularly communicate with your collaborator to ensure you're both aware of recent changes.
Ensure that Unreal Engine's intermediate and saved folders are properly ignored in your .gitignore file.
